spec/computer_spec.rb

```diff
@@ -1,75 +1,87 @@
-require 'computer.rb'
-require 'game.rb'
-require 'board.rb'
+require File.join(File.dirname(__FILE__), '..', 'lib', 'computer')
+require File.join(File.dirname(__FILE__), '..', 'lib', 'game')
+require File.join(File.dirname(__FILE__), '..', 'lib', 'board')
+
 
 describe Computer do
   let(:computer) { Computer.new('O') }
   let(:board) { Board.new }
   let(:referee) { Referee.new }
 
+  def set_board_values(board, hash_values)
+    board.tokens = hash_values
+  end
+
   it "has an enemy token" do
     expect(computer.enemy_token).to eq('X')
   end
 
   it "blocks human win" do
-    board.set_token_at(1, 'X')
-    board.set_token_at(4, 'O')
-    board.set_token_at(2, 'X')
+    set_board_values(board, {
+      0 => 'X', 1 => 'X', 2 => '3',
+      3 => 'O', 4 => '5', 5 => '6',
+      6 => '7', 7 => '8', 8 => '9'
+    })
```

Comment:
un hash es lo mas legible?

Reply:
No, I've just implemented the structure you recommended in my last review

Reply:
para board internamente sí, pedi un hash, pero esto no quiere decir que en tus pruebas debas usar un hash – el chiste es que esa decisión no impacte la legibilidad de otras áreas de tu sistema